app/src/main/java/com/serwylo/lexica/game/Game.java

```diff
@@ -86,6 +88,8 @@
 	private LinkedList<String> wordList;
 	private LinkedHashSet<String> wordsUsed;
 	private int wordCount;
+	private final int[] wordCountsByLength = new int[25];
+	private final int[] maxWordCountsByLength = new int[25];
```

Comment:
I think a [`SparseIntArray`](https://developer.android.com/reference/android/util/SparseIntArray.html) might be better for this job. Although it wont make a real practical difference, it means a couple less magic numbers. For example, if someone wants to allow the board to be a 6x6 board in the future then it is not clear that this (theoretical) limit on the word length (which will likely never be hit) needs to change.